app/federation/server/startup/registerCallbacks.js

```diff
@@ -0,0 +1,24 @@
+import { registerCallback } from '../lib/callbacks';
+import { definition as afterAddedToRoomDef } from '../hooks/afterAddedToRoom';
```

Comment:
if you avoid `export default` you could import like `import * as afterCreateDirectRoomDef ....`